source/tests/e2e/test_app.py

```diff
@@ -57,5 +56,17 @@ def test_everything(browser, test_environment, stack_resources):
     rows = browser.find_elements("xpath", xpath)
     assert len(rows) > 0
 
+    # open Step 2
+    browser.find_element(By.ID, "step2").click()
+    wait.until(EC.presence_of_element_located((By.ID, "dataset_type_options")))
```

Comment:
NP: i wonder if a fixed amount of timeout should be set like [this](https://selenium-python.readthedocs.io/waits.html?highlight=wait.until#explicit-waits). This way it's not freezing the whole process, waiting for element to be available. Is also good to know why is taking long for element to show up, for good user experience.

Reply:
I am using a fixed timeout. Not sure what you are suggesting here. Also automating the process of troubleshooting slow UX is outside the scope of this PR.

Reply:
ah ok, i see, please disregard.